test/recommended.js

```diff
@@ -85,16 +86,26 @@ describe('recommended config', () => {
             }
         `);
 
-        const { messages } = report.results[0];
-        assert.strictEqual(messages.length, 2);
-        assert.strictEqual(messages[0].ruleId, '@lwc/lwc/no-dupe-class-members');
-        assert.strictEqual(messages[1].ruleId, '@lwc/lwc/no-dupe-class-members');
+        const { messages } = results[0];
+        const isEslint7 = !!eslint.CLIEngine;
+        const expected = isEslint7
+            ? ['@lwc/lwc/no-dupe-class-members', '@lwc/lwc/no-dupe-class-members']
+            : [
+                  '@lwc/lwc/no-dupe-class-members',
+                  'no-dupe-class-members',
+                  '@lwc/lwc/no-dupe-class-members',
+                  'no-dupe-class-members',
```

Comment:
What do you think about disabling the `@lwc/lwc/no-dupe-class-members` rule when running ESLint 8? For example, you can import `eslint` from within the `recommended.js` and check if the version is greater than `8`.

Reply:
@pmdartus OK, I'm fine with checking the explicit version rather than sniffing it.

Reply:
The only issue there is that we need an extra devDependency on `semver`, but it's fine.

Reply:
I just realized I totally misread your comment. I'm not sure what the impact is of disabling the `@lwc/lwc/no-dupe-class-members` rule when ESLint 8 is running. Is that potentially a breaking change?

Reply:
I don't see this as a breaking change.

Reply:
Should we cover that in another PR?

Reply:
Ok let's do this.